packages/app/src/app/index.js

```diff
@@ -120,15 +121,17 @@ function boot() {
 
     try {
       render(
-        <Provider {...controller.provide()}>
-          <ApolloProvider client={client}>
-            <ThemeProvider theme={theme}>
-              <Router history={history}>
-                <App />
-              </Router>
-            </ThemeProvider>
-          </ApolloProvider>
-        </Provider>,
+        <Cerebral.Provider value={controller.provide()}>
```

Comment:
Interesting, why do we do this? Aren't we setting the context twice now?

Reply:
Technically yes, we are using two providers here. This seems to be the recommended approach to migrating to mobx-react-lite, as this new provider uses the react context api, and the old way does it different somehow. Basically to use `useContext` we have to do it this way. Doesn't appear to be a problem at the moment, but I'm happy to do further tests before we commit to this.

Reply:
This has been re-addressed in 519b214, as now it's only calling `controller.provide()` once and passing it's values to our providers. Once the whole refactor is complete, we can remove the old provider entirely.